action.yml

```diff
@@ -27,11 +27,53 @@ inputs:
     description: 'Apply custom chapters for PRs without linked issues.'
     required: false
     default: 'true'
+  verbose:
+    description: 'Print verbose logs.'
+    required: false
+    default: 'false'
+outputs:
+  release-notes:
+    description: 'Generated release notes.'
+    value: ${{ steps.release-notes-generator.outputs.release-notes }}
 
 branding:
   icon: 'book'
   color: 'yellow'
 
 runs:
-  using: 'node20'
-  main: 'dist/index.js'
+  using: 'composite'
+  steps:
+    - name: Install Python dependencies
+      run: |
+        python3 -m pip install --upgrade pip
```

Comment:
This should not be a part of the composite action. Let the user figure it out.

Reply:
Removed.